### PR TITLE
Added Slovak translation

### PR DESCRIPTION
**Pull request by matejbadin**
Slovak translation added.

### CODE REVIEW

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+msgid "reply"
+msgstr "odpovedať"
+
+#: templates/comments/deleted.html:4
+msgid "Thanks for removing"
+msgstr "Ďakujeme za odstránenie"
+
+#: templates/comments/deleted.html:12
+msgid "Thanks for removing the comment"
+msgstr "Ďakujeme za odstranění komentára"
```
Comment (ZoltanOnody) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: this one is in czech

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+"    "
+
+#: templates/comments/deleted.html:20 templates/comments/flagged.html:20
+#: templates/comments/posted.html:24
+msgid "Back to the article"
+msgstr "Naspäť k diskusii"
+
+#: templates/comments/flagged.html:4
+msgid "Thanks for flagging"
+msgstr "Děkujeme za označkování"
```
Comment (ZoltanOnody) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: this one is in czech

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+#: templates/comments/preview.html:10
+msgid "Please correct the error below"
+msgid_plural "Please correct the errors below"
+msgstr[0] "Prosím opravte nasledujúcu chybu"
+msgstr[1] "Prosím opravte nasledujúce chyby"
+
+#: templates/comments/preview.html:17
+msgid "Post your comment"
+msgstr "Odoslať komentář"
```
Comment (ZoltanOnody) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: this one is in czech

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+#: templates/comments/preview.html:20
+msgid "Or make changes"
+msgstr "Alebo spravte zmeny"
+
+#: templates/fluent_comments/templatetags/ajax_comment_tags.html:2
+msgid "cancel reply"
+msgstr "zrušit odpověď"
```
Comment (ZoltanOnody) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: this one is in czech

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+#: templates/fluent_comments/templatetags/ajax_comment_tags.html:4
+msgid "Please wait . . ."
+msgstr "Počkajte, prosím . . ."
+
+#: templates/fluent_comments/templatetags/ajax_comment_tags.html:6
+msgid "Your comment has been posted!"
+msgstr "Váš komentář byl publikován!"
```
Comment (ZoltanOnody) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: this one is in czech

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -158,7 +158,7 @@ msgstr "Počkajte, prosím . . ."
 
 #: templates/fluent_comments/templatetags/ajax_comment_tags.html:6
 msgid "Your comment has been posted!"
-msgstr "Váš komentář byl publikován!"
+msgstr "Váš komentář bol publikovaný!"
```
Comment (ZoltanOnody) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: change "ř" to "r" in komentář

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2\n"
+
+#: admin.py:45
+msgid "Content"
+msgstr "Obsah"
+
+#: admin.py:48
+msgid "Account information"
+msgstr "Informace o účte"
```
Comment (koniiiik) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: Informácie

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+"\n"
+"      Thanks for taking the time to improve the quality of discussion on our "
+"site.<br/>\n"
+"      You will be sent back to the article...\n"
+"    "
+msgstr ""
+"\n"
+"      Vďaka, že ste si našli čas na zlepšenie kvality diskusie na našom "
+"webe.<br/>\n"
+"      Budete presmerovaný späť na diskusiu...\n"
```
Comment (koniiiik) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: Prefer gender-neutral formulations: „Presmerujeme vás späť na diskusiu...“

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+#: templates/comments/posted.html:13
+msgid ""
+"\n"
+"      We have received your comment, and posted it on the web site.<br/>\n"
+"      You will be sent back to the article...\n"
+"      "
+msgstr ""
+"\n"
+"      Dostali sme váš komentár a zobrazili sme ho na stránke.<br/>\n"
+"      Budete presmerovaný späť na diskusiu...\n"
```
Comment (koniiiik) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: Prefer gender-neutral formulations: „Presmerujeme vás späť na diskusiu...“

File: fluent_comments/locale/sk/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -0,0 +1,169 @@
...
+msgstr "Náhľad komentára"
+
+#: templates/comments/comment.html:24
+msgid "Anonymous"
+msgstr "Anonym"
+
+#: templates/comments/comment.html:27
+#, python-format
+msgid "on %(submit_date)s"
+msgstr "poslal %(submit_date)s"
```
Comment (koniiiik) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: „odoslané %(submit_date)s“ or even „pridané %(submit_date)s“

File: fluent_comments/locale/sk/LC_MESSAGES/django.po (hunk 4 above)
Comment (koniiiik) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: Also, the “t” is lacking a caron, so „zrušiť odpoveď“.

File: fluent_comments/locale/sk/LC_MESSAGES/django.po (hunk 3 above)
Comment (koniiiik) [inline fluent_comments/locale/sk/LC_MESSAGES/django.po]: As @ZoltanOnody pointed out, the correct form would be „Odoslať komentár“ without the caron on “r”.

**Comment by vdboor** [comment]
Thanks for the work @matejbadin  and review @koniiiik @ZoltanOnody !

**Pull request merged**